Update test instructions in to-test.md for Jetpack 15.3 (#46042)
Automattic/jetpack: projects/plugins/jetpack/to-test.md

### Social
Related PRs: #45970, #45939

#### Testing Steps:

##### Emoty states updates
- Go to the editor and open Jetpack sidebar
- Confirm that you see the auto-share toggle changed to checkbox with the updated label
- Confirm that the checkbox and the description are visible
- Login as another user (author or admin)
- Go to the editor again
- Confirm that you see the empty state to connect the wp.com account first
- Connect the wp.com account
- Confirm that now you see the empty state to connect the Social media accounts – like in the pull request description

##### Updates to the connections list
- Go to the post editor
- Open Jetpack sidebar
- Confirm that the connection toggles list is replaced with the updated vertical list with icons and labels.
- Confirm that the UI matches the screenshots from the pull request
- Confirm that the label for connections management button is changed to “Add a new account”
- Toggle the connections ON/OFF
- Confirm that it works fine
- Try to share a post with some connections OFF
- Confirm that the post is shared only to the accounts which were enabled.

### Blocks
Related PRs: #45967, #45900, #45776

#### Testing Steps:

##### Video block

> - Set up a test site with Jetapck and Gutenberg 21.4 or later

Reviewer: ```suggestion
- Set up a test site with Jetpack and Gutenberg 21.4 or later
```

> - Connect Jetpack and activate the videopress module

Reviewer 2: ```suggestion
- Connect Jetpack and activate the VideoPress module
```

- The video block should be accessible with both Jetpack and VideoPress active.

##### Gif block
- Go to posts
- Create a new post and add a gif block
- Type a query and select any other gif than the first one
- Save the post
- The selection of the user should be properly applied

##### Gdoc block preview
- Create a new post / page

> - Add a google slides block to it

Reviewer: ```suggestion
##### Google Sheets and Google Slides block preview
- Ensure beta blocks are enabled
- Create a new post / page
- Add a Google Sheets or Google Slides block to it
```

> - Paste in the share URL of a google slides doc

Reviewer 2: ```suggestion
- Paste in the share URL of a Google Sheets or Google Slides document
```

- Save the post / page

> - View the front-end, the doc should render.

Reviewer 2: ```suggestion
- View the post on the front-end and verify the document preview renders.
```
